nano/node/rpc.cpp

```diff
@@ -3314,6 +3314,53 @@ void nano::rpc_handler::unchecked_keys ()
 	response_errors ();
 }
 
+void nano::rpc_handler::unopened ()
+{
+	rpc_control_impl ();
```

Comment:
All lines between rpc_control_impl () & response_errors () in `if  (!ec) { ... }`
Other than that, LGTM